lib/awful/mouse/init.lua

```diff
@@ -25,8 +25,6 @@ local capi =
     mousegrabber = mousegrabber,
 }
 
-local finder = require("awful.mouse.finder")
```

Comment:
Fun fact: I noticed all of this because Luacheck was complaining about the above line (variable `finder` is assigned a value that is never used).